README.md

```diff
@@ -157,6 +178,29 @@ Typically Vagrant box deployments encounter one of few problems:
 
     uaa is not functioning, try steps above
 
+* volumes don't get mounted when suspending/resuming the box
+
+  For now only `vagrant stop` and then `vagrant up` fixes it.
+
+* When restarting the box with either `vagrant reload` or `vagrant stop/up` some
+  pods never come up automatically. You have to do a `make stop` and then
+  `make run` to bring this up.
+
+* Pulling images during any of `vagrant up` or `make vagrant-prep` or `make docker-deps`
+  fails.
+
+  Make sure you have enabled ipv4 ip fowarding.
+
+  Temporarily:
+
+  ```echo "1" | sudo tee /proc/sys/net/ipv4/ip_forward```
```

Comment:
Ah, if it's required on the host then it could make sense. In that case we'd have to make that part clearer. `docker.service` should have fixed it, but it's possible that (since the host is long-running) it got reset by something else (e.g. `systemctl restart NetworkManager.service`).